pinot-tools/src/main/resources/examples/batch/starbucksStores/starbucksStores_offline_table_config.json

```diff
@@ -11,16 +11,18 @@
   },
   "tenants": {
   },
+  "fieldConfigList": [
+    {
+      "name": "location_st_point",
+      "encodingType":"RAW",
+      "indexType":"H3",
+      "properties": {
```

Comment:
this does not match the H3ColumnIndex as BaseConfig, right?

Reply:
It uses the properties to construct.